tuxemon/core/config.py

```diff
@@ -171,6 +186,17 @@ def get_defaults():
             ("debug_logging", True),
             ("debug_level", "error")
         ))),
+        ("controls", OrderedDict((
```

Comment:
this should be reversed to allow many keys to do the same thing (lshift and rshift):
```python

        ("controls", OrderedDict((
            ("lshift", "player_b"),
            ("rshift", "player_b"),
            ...
```

Reply:
config would be:
```
[controls]
W = UP
S = DOWN
```
I'm note sure if including "player_"... is important or not, but the code is more simple if removed.  your call there.

Reply:
Yeah, this makes sense. I'll remove "player_" because it makes more sense for a user to simply type in the button they want rather than player_"<insert button>". Thanks for the suggestions!